index.tt

```diff
@@ -1,157 +1,256 @@
-      to package and configuration management. Built on top of the
-      Nix package manager, it is completely declarative, makes
-      upgrading systems reliable, and has
-      <a href="[%root%]nixos/about.html">many other advantages</a>.</p>
-    
-      <div class="learn-more">
-        <a class="btn btn-success" href="[%root%]nixos/">
-          <i class="fa fa-chevron-right"></i> Learn more about NixOS
-        </a>
-      </div>
-    </div>
-  </div></div>
-  
-  <hr />
-  
-  <h2>More about NixOS</h2>
-  
-  <div class="row-fluid more-about">
-    <div class="span4">
-      <h3>Nixpkgs</h3>
-      <p>
-        The Nix Packages collection (Nixpkgs) is the ever growing
-        packages collection for the Nix package manager, released
-        under a permissive MIT/X11 license.
-      </p>
-      <a class="learn-more" href="[%root%]nixpkgs">
-        More about Nixpkgs
-        <i class="fa fa-angle-right"></i>
-      </a>
-    </div>
-    <div class="span4">
-      <h3>NixOps</h3>
-      <p>
-        NixOps is a tool for deploying sets of NixOS Linux machines,
-        bare-metal or to the cloud. It extends NixOS’s declarative
-        approach to system configuration management to networks and
-        provisioning.
-      </p>
-      <a class="learn-more" href="https://github.com/NixOS/nixops">
-        More about NixOps
-        <i class="fa fa-angle-right"></i>
-      </a>
-    </div>
-    <div class="span4 and-more">
-      <h3>About the community</h3>
-      <p>
-        Sometimes, it helps to talk with someone.  Our community
-        is there whether you want to help or get support.
-      </p>
-      <ul>
-        <li>
-          <a href="[%root%]nixos/community.html">
-            Get in touch
-            <i class="fa fa-angle-right"></i>
-          </a>
-        </li>
-        <li>
-          <a href="[%root%]learn.html">
-            Learning resources
-            <i class="fa fa-angle-right"></i>
-          </a>
-        </li>
-      </ul>
-    </div>
+<div class="row-fluid">
+  <div class="span6">
+    <h1>Reproducible builds and deployments.</h1>
+    <p class="lead">
+      <strong>Nix</strong> is a powerful package manager for Linux and other
+      Unix systems that makes package management reliable and reproducible.
+      Share your development and build environments across different machines.
+    </p>
+    <p class="lead">
+      <strong>NixOS</strong> is a Linux distribution with a unique approach to
+      package and configuration management. Built on top of the Nix package
+      manager, it is completely declarative, makes upgrading systems reliable,
+      and has <a href="[% root %]features.html">many other advantages</a>.
+    </p>
+    <a class="btn btn-large btn-success" href="[% root %]download.html">Download</a>
+    <a class="btn btn-large" href="[% root %]learn.html">Get started</a>
+  </div>
+
+  <div class="span6">
+    <!-- TODO: Add a proper video / picture of NixOS in action! -->
```

Comment:
@garbas can you create a low effort version of it? maybe just the first example. then people see it there, think of cool, and then see the example at the bottom again and might understand how awesome that actually is

don't let this block merging this PR